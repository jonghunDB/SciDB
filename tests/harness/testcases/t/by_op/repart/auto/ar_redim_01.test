# SDB-5110.  An optimizer-inserted redimension (for purposes of
# repartitioning to a schema whose high bound changed) can't find the
# name of a destination attribute in the source array.

--setup
--test
store(build(<a:int64>[x=0:2:0:3], x+10), A)
store(build(<v:int64>[x=0:0:0:1], 42), A)

--cleanup
remove(A)
