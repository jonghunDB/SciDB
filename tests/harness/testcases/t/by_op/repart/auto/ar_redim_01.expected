{x} a
{0} 10
{1} 11
{2} 12

{x} a
{0} 42

Query was executed successfully

